<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpoLua</string>
  </property>
  <widget class="OplScreenWidget" name="screen">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpenRecent">
     <property name="title">
      <string>Open Recent</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuOpenRecent"/>
    <addaction name="actionWelcome"/>
    <addaction name="actionInstall"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuInterpreter">
    <property name="title">
     <string>Interpreter</string>
    </property>
    <addaction name="actionMenu"/>
    <addaction name="actionRestart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionSeries5"/>
    <addaction name="actionSeries7"/>
    <addaction name="actionGeofoxOne"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInterpreter"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open OPL App...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionWelcome">
   <property name="text">
    <string>Open 'Welcome'</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+L</string>
   </property>
  </action>
  <action name="actionMenu">
   <property name="text">
    <string>Press 'Menu' key</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Esc</string>
   </property>
  </action>
  <action name="actionSeries5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Series 5</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+5</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
  </action>
  <action name="actionSeries7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Series 7</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+7</string>
   </property>
  </action>
  <action name="actionGeofoxOne">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Geofox One</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+1</string>
   </property>
  </action>
  <action name="actionInstall">
   <property name="text">
    <string>Install SIS file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OplScreenWidget</class>
   <extends>QLabel</extends>
   <header>oplscreenwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
